.factorypath

```diff
@@ -0,0 +1,55 @@
+<factorypath>
```

Comment:
Can you delete this file?

Reply:
These files keep coming back whenever I try switching branches. So I just added them to my .gitignore.